lib/uktt/export_chapter_pdf.rb

```diff
@@ -530,13 +532,13 @@ def update_anti_dumpings(v2_commodity)
         description += delimiter + additional_code.attributes.formatted_description
       end
 
-      unless description == ''
-        commodity_item_id = v2_commodity.data.attributes.goods_nomenclature_item_id
-        geographical_area_id = measure.relationships.geographical_area.data.id
-        @anti_dumpings[commodity_item_id] ||= {}
-        @anti_dumpings[commodity_item_id][geographical_area_id] ||= {}
-        @anti_dumpings[commodity_item_id][geographical_area_id][additional_code&.attributes&.code || ''] ||= description
-      end
+      next if description == ''
```

Comment:
```suggestion
      next if description.blank?
```

Reply:
Think blank is an ActiveSupport thing https://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark

Reply:
This gem doesn't actually bundle ActiveSupport